src/plone/restapi/serializer/dxfields.py

```diff
@@ -11,7 +11,6 @@
 from zope.interface import Interface
 from zope.schema.interfaces import IField
 
-
```

Comment:
Please do not change that file as the change is not related to the added functionality.